pkg/inspector/image-inspector.go

```diff
@@ -337,7 +338,7 @@ func (i *defaultImageInspector) pullImage(client *docker.Client) error {
 			}
 			go decodeDockerResponse(parsedErrors, reader)
 
-			if err := client.PullImage(imagePullOption, auth); err != nil {
+			if err = client.PullImage(imagePullOption, auth); err != nil {
```

Comment:
this seems to be changing the behavior of `authErr` but looking at the old code I don't see where `authErr` was ever set anyway.  Was this a bug?